src/templates/outstream/porvata-template.js

```diff
@@ -0,0 +1,123 @@
+import { context, events, Porvata, slotTweaker } from '@wikia/ad-engine';
+import * as videoUserInterface from '../interface/video';
+
+export const DEFAULT_VIDEO_ASPECT_RATIO = 640 / 360;
+export const IMA_VPAID_INSECURE_MODE = 2;
+
+export class PorvataTemplate {
+	static getName() {
+		return 'porvata3';
+	}
+
+	static getDefaultConfig() {
+		return {
+			isFloatingEnabled: true,
+			inViewportOffsetTop: 0,
+			inViewportOffsetBottom: 0
+		};
+	}
+
+	constructor(adSlot) {
+		this.adSlot = adSlot;
+		this.config = context.get('templates.porvata3');
+	}
+
+	init(params) {
+		const slotName = this.adSlot.getSlotName();
+		const isInsecureMode = params.vpaidMode === IMA_VPAID_INSECURE_MODE;
```

Comment:
It can be assigned to `this` and reused in line 47.

Reply:
Done